core/loader/json-feed.ts

```diff
@@ -91,19 +91,34 @@ function validate<ValidatedType>(
   return true
 }
 
+const IMG_URL_REGEXP = /<img[^>]+src="?([^"\s]+)"?\s*\/?>/g
+
 function parsePosts(text: TextResponse): OriginPost[] {
   let parsedJson = text.parseJson()
   if (!validate<JsonFeed>(parsedJson, JSON_FEED_VALIDATORS)) return []
 
-  return parsedJson.items.map(item => ({
-    full: (item.content_html || item.content_text) ?? undefined,
-    intro: item.summary ?? undefined,
-    media: [],
-    originId: item.id,
-    publishedAt: toTime(item.date_published) ?? undefined,
-    title: item.title,
-    url: item.url ?? undefined
-  }))
+  return parsedJson.items.map(item => {
+    const full = (item.content_html || item.content_text) ?? undefined
+    const allImages: string[] = [item.banner_image ?? '', item.image ?? '']
+
+    if (full) {
+      let match: RegExpExecArray | null
+      while ((match = IMG_URL_REGEXP.exec(full))) {
+        const url = match[1]
+        url && allImages.push(url)
```

Comment:
```suggestion
        if (url) allImages.push(url)
```

This one-line if-statement if more easy to read

Reply:
Updated